bokehjs/src/coffee/embed.ts

```diff
@@ -48,6 +49,10 @@ declare interface Jupyter {
 
 declare var Jupyter: Jupyter | undefined
 
+export const kernels: {[key: string]: any} = {}
+
+export {Receiver}
```

Comment:
Is this for Holoviews? AFAICT jupyterlab_bokeh doesn't require it.

Reply:
That's right, HoloViews uses it to make a generic message handler which should work outside Jupyter as well. Would be really helpful if this can be public.